frontend/components/footer/index.jsx

```diff
@@ -13,9 +15,17 @@ const Footer = ({ data }) => {
     } = data;
 
     const { title, description, url } = footer_about ? footer_about : false;
+    const [ref, inView] = useInView({
+      threshold: 0.1
+    });
     return (
-      <footer className="container">
-        <div className="grid lg:grid-cols-3 grid-cols-1">
+      <footer ref= {ref} className="container">
+        <motion.div
+          initial={{ y: -200, opacity: 0 }}
```

Comment:
I think it will be done ( comes from the bottom ) by reflecting Y from **-200** To **200**
and add `overflow-hidden` class, side the container class at the footer element